code/datums/gamemode/objectives/target/sacrifice.dm

```diff
@@ -2,6 +2,7 @@
 
 /datum/objective/target/assassinate/sacrifice
 	name = "Sacrifice <target>"
+	bad_assassinate_targets = list("Trader", "Merchant") // Bird creep !
```

Comment:
The Trader one isn't actually necessary because I believe the reason Merchant was being picked was due to having a loyalty implant. Also I doubt would work otherwise because bad_assassinate_targets checks against mind.assigned_role which should be "Trader", you would need to compare against mind.role_alt_title for merchants.

So in this very specific case where it doesn't matter it's okay, but in the future remember that alt titles are different than job titles